index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link rel="stylesheet" href="google.css">
+</head>
+
+<body>
+    <div class="main">
+        <div class="navbar">
+            <ul>
+                <li><a href="#">Gmail</a></li>
+                <li><a href="#">Images</a></li>
+                <li><a href="#"><img src="Images\dots.png" alt="grid"></a></li>
+                <li><a href="#"><img src="Images\user-profile-icon-free-vector.jpg" alt="Profile"></a></li>
+            </ul>
+        </div>
+        <div class="title">
+            <img src="Images\image2.png" alt="Google">
+        </div>
+        <div class="search">
+            <div class="searchbar">
+                <div class="content">
+                    <textarea class="text" rows="1" cols="1"></textarea>
+                    <img class="search_img" src="Images\free-search-icon-2903-thumb.png" alt="search">
+                    <img class="google_lens" src="Images\Google_Lens_Icon.svg.png" alt="Search with image">
+                </div>
+            </div>
+            <div class="buttons">
+                <button class="buttons_search">Google Search</button>
+                <button class="buttons_search">I'm Feeling Lucky</button>
+            </div>
+        </div>
+        <div class="language">
+            <div class="language_choices">
+                <div class="language_choices_text">
+                    Google offered in:
+                </div>
+                <div>
+                    <ul>
+                        <li><a href="#">हिन्दी</a></li>
+                        <li><a href="#">বাংলা</a></li>
+                        <li><a href="#">తెలుగు</a></li>
+                        <li><a href="#">मराठी</a></li>
+                        <li><a href="#">தமிழ்</a></li>
+                        <li><a href="#">ગુજરાતી</a></li>
+                        <li><a href="#">ಕನ್ನಡ</a></li>
+                        <li><a href="#">മലയാളം</a></li>
+                        <li><a href="#">ਪੰਜਾਬੀ</a></li>
```

Comment:
There is no underline on any of these link. It is preferred to have underline on link, it give intuition to the use that the particular text is a link